app/Domain/Timesheets/Templates/editTime.tpl.php

```diff
@@ -98,7 +98,7 @@
 
 </select><br />
 <label for="date"><?php echo $tpl->__('label.date')?></label> <input type="text" autocomplete="off"
-    id="datepicker" name="date" value="<?php echo format($values['date'], 'short'); ?>" size="7" />
+    id="datepicker" name="date" value="<?php echo format($values['date'], "", \Leantime\Core\Support\FromFormat::DbDate)->date(); ?>" size="7" />
```

Comment:
You should be able to add an `use` statement in the top of the file and then use the short and more readable version here.

Reply:
We could also use named parameters here? To remove the empty string in these calls.